src/util/useLibraryOptions.ts

```diff
@@ -20,19 +22,23 @@ interface IUseLibraryOptions {
 
 export default function useLibraryOptions(): IUseLibraryOptions {
     const [searchQuery, setSearchQuery] = useQueryParams({
+        downloaded: BooleanParam,
         unread: BooleanParam,
         query: StringParam,
     });
-    const { unread, query } = searchQuery;
+    const { downloaded, unread, query } = searchQuery;
+    const setDownloaded = (newDownloaded: NullAndUndefined<boolean>) => {
+        setSearchQuery(Object.assign(searchQuery, { downloaded: newDownloaded }), 'replace');
+    };
     const setUnread = (newUnread: NullAndUndefined<boolean>) => {
         setSearchQuery(Object.assign(searchQuery, { unread: newUnread }), 'replace');
     };
     const setQuery = (newQuery: NullAndUndefined<string>) => {
         setSearchQuery(Object.assign(searchQuery, { query: newQuery }), 'replace');
     };
     // eslint-disable-next-line eqeqeq
-    const active = !(unread == undefined);
+    const active = !(unread == undefined) && !(downloaded == undefined);
```

Comment:
```suggestion
    const active = !(unread == undefined) || !(downloaded == undefined);
```
It's active when there's a value for the unread filter or the dowloaded filter